### PR TITLE
Review of python new package metadata

### PR DESCRIPTION
**Pull request by czubair**

### CODE REVIEW

File: _data/releases/latest/python-packages.csv [hunk 1]
```diff
@@ -381,10 +381,10 @@
 "mltable","1.3.0","","Machine Learning - Table","Machine Learning","NA","NA","NA","","false","","","","","","","","","machine-learning","",""
 "azure-purview-administration","","1.0.0b1","Purview Administration","Purview","purview","NA","NA","","false","","","","","","","","","","",""
 "text-analytics","1.0.2","","Text Analytics","Cognitive Services","NA","NA","NA","","false","","","","","","","","","","",""
-"apiview-stub-generator","","0.3.7","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"azure-ai-mlmonitoring","","0.1.0a3","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"azure-iot-deviceprovisioning","","1.0.0b1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"azure-pylint-guidelines-checker","","0.0.8","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
+"apiview-stub-generator","","0.3.7","Stub Generator","API View","NA","NA","NA","","false","","","","","","","","","","",""
```
Comment (ronniegeraghty) [inline _data/releases/latest/python-packages.csv:384]: @czubair, is this a package for API? Meaning it's not something our customers would use?

File: _data/releases/latest/python-packages.csv [hunk 2]
```diff
@@ -381,10 +381,10 @@
...
+"azure-ai-mlmonitoring","","0.1.0a3","Machine Learning - Monitoring","Machine Learning","NA","NA","NA","","false","","","","","","","","","","",""
+"azure-iot-deviceprovisioning","","1.0.0b1","IoT Device Provisioning","IoT","NA","NA","NA","","false","","","","","","","","","","",""
+"azure-pylint-guidelines-checker","","0.0.8","Pylint Guidelines Checker","Pylint","NA","NA","NA","","false","","","","","","","","","","",""
```
Comment (ronniegeraghty) [inline _data/releases/latest/python-packages.csv:387]: @czubair, same as last comment. Is this a tool for our internal use and we don't expect customers to use it?

File: _data/releases/latest/python-packages.csv (hunk 1 above)
Comment (czubair) [reply _data/releases/latest/python-packages.csv:384]: yes, I believe so. but it still shows up with a "needs review"label that at the very least, needs removed.

Comment (czubair) [reply _data/releases/latest/python-packages.csv:384]: @ronniegeraghty and @weshaggard how should internal packages be marked?

Comment (weshaggard) [reply _data/releases/latest/python-packages.csv:384]: This one is actually a tool for APIView so I don't expect customers would use it.

Comment (weshaggard) [reply _data/releases/latest/python-packages.csv:384]: @ronniegeraghty I believe you have been marking these as tools correct? We could also consider hiding these packages.

Comment (xiangyan99) [reply _data/releases/latest/python-packages.csv:384]: This is an internal tool.

File: _data/releases/latest/python-packages.csv (hunk 2 above)
Comment (xiangyan99) [reply _data/releases/latest/python-packages.csv:387]: Same. This is an internal tool. "azure-pylint-guidelines-checker"

File: _data/releases/latest/python-packages.csv (hunk 1 above)
Comment (czubair) [reply _data/releases/latest/python-packages.csv:384]: @ronniegeraghty I like the idea of hiding them. I ma fine with marking these as tools as well. Can you please complete one as an example so I can get the details right?

Comment (czubair) [inline _data/releases/latest/python-packages.csv:384]:
```suggestion
"apiview-stub-generator","","0.3.7","Stub Generator","API View","NA","NA","NA","tool","false","","","","","","","","","","",""
```

File: _data/releases/latest/python-packages.csv [hunk 3]
```diff
@@ -381,10 +381,10 @@
...
-"apiview-stub-generator","","0.3.7","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"azure-ai-mlmonitoring","","0.1.0a3","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"azure-iot-deviceprovisioning","","1.0.0b1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"azure-pylint-guidelines-checker","","0.0.8","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
+"apiview-stub-generator","","0.3.7","Stub Generator","API View","NA","NA","NA","tool","false","","","","","","","","","","",""
+"azure-ai-mlmonitoring","","0.1.0a3","Machine Learning - Monitoring","Machine Learning","NA","NA","NA","","false","","","","","","","","","","",""
+"azure-iot-deviceprovisioning","","1.0.0b1","IoT Device Provisioning","IoT","NA","NA","NA","","false","","","","","","","","","","",""
+"azure-pylint-guidelines-checker","","0.0.8","Pylint Guidelines Checker","Pylint","NA","NA","NA","","false","","","","","","","","","","",""
```
Comment (czubair) [inline _data/releases/latest/python-packages.csv:387]:
```suggestion
"azure-pylint-guidelines-checker","","0.0.8","Pylint Guidelines Checker","Pylint","NA","NA","NA","tool","false","","","","","","","","","","",""
```

**Comment by czubair** [comment]
@xiangyan99 could you have a look at this?

File: _data/releases/latest/python-packages.csv [hunk 4]
```diff
@@ -381,10 +381,10 @@
...
 "text-analytics","1.0.2","","Text Analytics","Cognitive Services","NA","NA","NA","","false","","","","","","","","","","",""
-"apiview-stub-generator","","0.3.7","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"azure-ai-mlmonitoring","","0.1.0a3","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"azure-iot-deviceprovisioning","","1.0.0b1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"azure-pylint-guidelines-checker","","0.0.8","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
+"apiview-stub-generator","","0.3.7","Stub Generator","API View","NA","NA","NA","tool","false","","","","","","","","","","",""
+"azure-ai-mlmonitoring","","0.1.0a3","Machine Learning - Monitoring","Machine Learning","NA","NA","NA","","false","","","","","","","","","","",""
+"azure-iot-deviceprovisioning","","1.0.0b1","IoT Device Provisioning","IoT","NA","NA","NA","","false","","","","","","","","","","",""
```
Comment (xiangyan99) [inline _data/releases/latest/python-packages.csv:386]: Isn't it a new library?

File: _data/releases/latest/python-packages.csv [hunk 5]
```diff
@@ -381,10 +381,10 @@
...
 "azure-purview-administration","","1.0.0b1","Purview Administration","Purview","purview","NA","NA","","false","","","","","","","","","","",""
 "text-analytics","1.0.2","","Text Analytics","Cognitive Services","NA","NA","NA","","false","","","","","","","","","","",""
-"apiview-stub-generator","","0.3.7","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"azure-ai-mlmonitoring","","0.1.0a3","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"azure-iot-deviceprovisioning","","1.0.0b1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"azure-pylint-guidelines-checker","","0.0.8","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
+"apiview-stub-generator","","0.3.7","Stub Generator","API View","NA","NA","NA","tool","false","","","","","","","","","","",""
+"azure-ai-mlmonitoring","","0.1.0a3","Machine Learning - Monitoring","Machine Learning","NA","NA","NA","","false","","","","","","","","","","",""
```
Comment (xiangyan99) [inline _data/releases/latest/python-packages.csv:385]: I have no idea what this is.

Should we just remove it?

File: _data/releases/latest/python-packages.csv (hunk 4 above)
Comment (xiangyan99) [reply _data/releases/latest/python-packages.csv:386]: Is it by design we have empty RepoPath here?

File: _data/releases/latest/python-packages.csv (hunk 5 above)
Comment (czubair) [reply _data/releases/latest/python-packages.csv:385]: No, we can leave it.

File: _data/releases/latest/python-packages.csv (hunk 4 above)
Comment (czubair) [reply _data/releases/latest/python-packages.csv:386]: @xiangyan99 there should be a repo path. @weshaggard for best way to find it.

Comment (weshaggard) [reply _data/releases/latest/python-packages.csv:386]: Yes this looks like it is newer library so we probably should add a repo path and setup type=client and new=true fields. 
```suggestion
"azure-iot-deviceprovisioning","","1.0.0b1","IoT Device Provisioning","IoT","iothub","NA","NA","client","true","","","","","","","","","","",""
```

**Comment by czubair** [comment]
@weshaggard can you help with the ci failure due to a broken link

**Comment by weshaggard** [comment]
Fixed in https://github.com/Azure/azure-sdk/pull/6399

**Review by xiangyan99 — APPROVED** [review]

**Pull request closed without merging**